spanner_orm/tests/field_test.py

```diff
@@ -104,6 +117,7 @@ def test_field_type_validate_type_error(
   @parameterized.parameters(
       (field.Boolean(), field.Boolean(), True),
       (field.Boolean(), field.String(), False),
+      (field.String(10), field.String(20), True),
```

Comment:
(nit) Mind adding `(field.String(), field.String(20), True),` too?

Reply:
Done.

Reply:
One of the strings should have a length, to make sure that one with a length is comparable to one without.